src/large_pages/node_large_page.cc

```diff
@@ -352,7 +352,7 @@ MoveTextRegionToLargePages(const text_region& r) {
     return -1;
   }
 
-  ret = madvise(tmem, size, MADV_HUGEPAGE);
+  ret = madvise(tmem, size, 14 /* MADV_HUGEPAGE */);
```

Comment:
??

Reply:
@devnexen see https://github.com/nodejs/node/pull/30954#pullrequestreview-332209376

Reply:
Sure but I thought it would be more appropriate to consider the feature unsupported if the system does have the flag.

Reply:
@devbexen if our releases are built on a system that doesn't have the flag then large pages will be unsupported for all those who download the binary, even if their system has the flag.

Reply:
Good point.